src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -375,14 +389,8 @@ extension ExposureSubmissionCoordinator {
 	func showWarnOthersScreen() {
```

Comment:
please check if this func might be private

Reply:
Refactored to private 👍